packages/test/src/test-testenvironment.ts

```diff
@@ -0,0 +1,149 @@
+import anyTest, { TestInterface } from 'ava';
+import { v4 as uuid4 } from 'uuid';
+import { TestWorkflowEnvironment, workflowInterceptorModules } from '@temporalio/testing';
+import { Worker } from '@temporalio/worker';
+import { WorkflowFailedError } from '@temporalio/client';
+import { assertFromWorkflow, sleep, raceActivityAndTimer } from './workflows/testenv-test-workflows';
+
+interface Context {
+  testEnv: TestWorkflowEnvironment;
+}
+
+async function withWorker<R>(worker: Worker, fn: () => Promise<R>): Promise<R> {
```

Comment:
Should we export this from `/testing`? Think it's going to be a common pattern we want people to follow, so it'll either be in our samples for people to copy-paste, or they import it.

Reply:
Good idea, not sure where to export this from though. Maybe this could be a method on the worker..